integrations/apache-camel/src/main/java/org/apache/plc4x/camel/Plc4XConsumer.java

```diff
@@ -84,22 +79,29 @@ public void setExceptionHandler(ExceptionHandler exceptionHandler) {
     }
 
     @Override
-    protected void doStart() throws InterruptedException, ExecutionException, TimeoutException {
-        PlcSubscriptionRequest request = new PlcSubscriptionRequest();
-        @SuppressWarnings("unchecked")
-        SubscriptionRequestCyclicItem subscriptionRequestCyclicItem = new SubscriptionRequestCyclicItem(dataType, address, TimeUnit.SECONDS, 3, this);
-        request.addItem(subscriptionRequestCyclicItem);
+    protected void doStart() throws InterruptedException, ExecutionException, TimeoutException, PlcException {
+        PlcSubscriber plcSubscriber = plcConnection.getSubscriber().orElseThrow(
+            () -> new PlcException("Connection doesn't support subscriptions."));
+        // TODO: Is it correct to only support one field?
```

Comment:
it depends how the camel uri is defined. In ads and modbus I omit query params so they can be used by camel